src/components/organisms/DropdownContent/DropdownContent.tsx

```diff
@@ -0,0 +1,121 @@
+import { FC } from 'react'
+import classNames from 'classnames'
+import { includes, map } from 'lodash'
+import CheckBoxInput from 'components/molecules/CheckBoxInput/CheckBoxInput'
+import Button, { AppearanceTypes } from 'components/molecules/Button/Button'
+import { SelectionControlsInputProps } from 'components/organisms/SelectionControlsInput/SelectionControlsInput'
+
+import classes from './styles.module.scss'
+
+type DropdownContentProps = SelectionControlsInputProps & {
+  isOpen?: boolean
+  selectedOptionObjects?: { label: string; value: string }[]
+}
+
+const DropdownContent: FC<DropdownContentProps> = ({
+  dropdownSize = 'l',
+  disabled,
+  isOpen,
+  error,
+  searchInput,
+  options,
+  multiple = false,
```

Comment:
**[blocker]**

1. For multi select the dropdown closes right after I make 1 change, making it very inconvenient to make multiple selections.

2. Currently it seems that the selected values from multiselect are saved to the form right away, which renders the "Filtreeri" button useless. Should confirm with Martine how that button is suppose to behave.

3. "Tühista" button doesn't seem to do anything other than closing the dropdown. Not sure if this is correct behaviour, should be confirmed with Martine.

When looking at the component, I would assume that all the selections I make are temporary and when I press the submit button, they will be stored, when I press the cancel button they will be reset to previous state. Although not sure how closing the dropdown by some other method should work in this case.